src/axom/quest/IntersectionShaper.hpp

```diff
@@ -1075,11 +1077,25 @@ class IntersectionShaper : public Shaper
   // Switch back to public. This is done here because the CUDA compiler
   // does not like the following template functions to be private.
 public:
+  /*!
+   * \brief Set the name of the material used to account for free volume fractions.
+   * \param name The new name of the material.
+   * \note This should not be called once any shaping has occurred.
```

Comment:
Should we add a check that shaping has not yet started?

Reply:
I added one using the number of cells member, which is set once the shaper has started its work.